src/datetime/tests.rs

```diff
@@ -873,6 +873,16 @@ fn test_datetime_from_local() {
     assert_eq!(datetime_west, datetime_utc.with_timezone(&timezone_west));
 }
 
+#[test]
+#[cfg(feature = "clock")]
+fn test_issue_651() {
```

Comment:
Please give this test a name that describes what it's testing for, and a comment/docstring containing a link to the issue.